apps/src/netsim/NetSimRouterNode.js

```diff
@@ -541,14 +624,13 @@ NetSimRouterNode.prototype.routeMessage_ = function (message, myWires) {
   var toAddress = message.payload.toAddress;
   if (toAddress === undefined) {
     //Malformed packet? Throw it away
-    logger.warn("Router encountered a malformed packet: " +
-    JSON.stringify(message.payload));
+    this.warn("Blocked malformed packet: " + message.payload);
     return;
   }
 
   if (toAddress === 0) {
```

Comment:
I don't think our spec includes packets addressed to the router, so special logic for address 0 can probably be omitted.